test/integration/provider/gcp.go

```diff
@@ -53,6 +53,23 @@ func getMachines(machineClass *v1alpha1.MachineClass, secretData map[string][]by
 	return machines, nil
 }
 
+func cleanUpOrphanResources(InstanceIds []string, VolumeIds []string, ctx context.Context, svc *compute.Service, project, zone string) (delErrInstanceId []string, delErrVolumeIds []string) {
+
+	for _, instanceId := range InstanceIds {
```

Comment:
why are `InstanceIds` and `VolumeIds` exported?

Reply:
will correct.

Reply:
done